WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -144,6 +159,7 @@ private enum Trigger
 	public ICommand PauseCommand { get; }
 
 	public ICommand StopCommand { get; }
+	public override string Title { get => throw new NotImplementedException(); protected set => throw new NotImplementedException(); }
```

Comment:
Looks hacky. If this solution was approved by UI team pls add a comment, and remove NotImplementedException in case of somehow the code gets here. Just return an empty string.

Reply:
Changed it lately, now it's better and working properly 👍